### PR TITLE
fix: fetch via authjs 4.6.2. bump version to 5.3.1

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "5.3.0",
+  "version": "5.3.1",
```
Comment (magizh-okta) [inline package.json:4]: Why this ?

Comment (aarongranick-okta) [reply package.json:4]: I am combining two PRs into one. Both would be changing the `package.json`. We can add other commits to the branch before we promote.

Comment (haishengwu-okta) [reply package.json:4]: not ideal as mess up the commit history a bit.

**Review by haishengwu-okta — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (aarongranick-okta) [reply package.json:4]: i can remove it if you want. i thought we were in a hurry

Comment (aarongranick-okta) [reply package.json:4]: i can't do separate commits on this PR because it is also modifying package.json

**Pull request closed without merging**